dev/cnf/oss_dependencies.maven

```diff
@@ -125,6 +125,7 @@ io.smallrye.common:smallrye-common-function:1.5.0
 io.smallrye.config:smallrye-config-common:2.0.2
 io.smallrye.config:smallrye-config-core:2.0.2
 io.smallrye.config:smallrye-config:2.0.2
+io.smallrye.reactive:smallrye-reactive-messaging-provider:1.0.9-SNAPSHOT
```

Comment:
Where is the other version? How come it wasn't listed to start with?

Reply:
it's in the `oss_source_dependencies.maven` file